examples/k8s/hz-go-service/README.md

```diff
@@ -0,0 +1,44 @@
+# Running Hazelcast Go Client on Kubernetes Example
+- This example shows how to run a sample proxy service which uses Hazelcast go client connected to the Hazelcast members in the same network and expose itself outside through k8s ingress.
+
+## Requirements
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+- [Docker](https://www.docker.com/get-started/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+## Start deployment
+Before the start, it is highly recommended using minikube on Docker Desktop. The guide supposes that Docker Desktop is ready and running properly. You can run it on other container and virtual machine environment, but it is not tested.
+1. Open a new terminal session and type specified command one by one.
+2. `minikube start --driver=docker`
+3. `minikube docker-env`
+4. `eval $(minikube -p minikube docker-env)`
+5. Open a new terminal session
+   1. `minikube addons enable ingress`
+   2. `sudo minikube tunnel`:
```

Comment:
Can we remove ":" in this line, or is it a part of the command?

Reply:
Edited, [9b1e80c](https://github.com/hazelcast/hazelcast-go-client/pull/743/commits/9b1e80c14c17b066528bdb55b557e87bbc0f16c3).